From 0f7029f88fff47c8ce70831085119433303fff37 Mon Sep 17 00:00:00 2001
From: sandra <sandra@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Thu, 21 May 2015 20:50:59 +0000
Subject: [PATCH] 2015-05-21  Sandra Loosemore  <sandra@codesourcery.com>

	Backport from mainline r223418:

	gcc/
	* config.gcc [powerpc*-*-linux*]: Allow --enable-targets=all
	to build a biarch toolchain again.


git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_9-branch@223503 138bc75d-0d04-0410-961f-82ee72b054a4
---
 gcc/ChangeLog  | 6 ++++++
 gcc/config.gcc | 1 +
 2 files changed, 7 insertions(+)

diff --git a/gcc/ChangeLog b/gcc/ChangeLog
index 3a4287a..97e7253 100644
--- a/gcc/ChangeLog
+++ b/gcc/ChangeLog
@@ -1,3 +1,9 @@
+2015-05-21  Sandra Loosemore  <sandra@codesourcery.com>
+
+	Backport from mainline r223418:
+	* config.gcc [powerpc*-*-linux*]: Allow --enable-targets=all
+	to build a biarch toolchain again.
+
 2015-05-19  Andreas Krebbel  <krebbel@linux.vnet.ibm.com>
 
 	* doc/invoke.texi: Add missing cpu types for march option: z196
diff --git a/gcc/config.gcc b/gcc/config.gcc
index d2bb780..7ade8e5 100644
--- a/gcc/config.gcc
+++ b/gcc/config.gcc
@@ -2285,6 +2285,7 @@ powerpc*-*-linux*)
 	maybe_biarch=${cpu_is_64bit}
 	case ${enable_targets} in
 	    *powerpc64*) maybe_biarch=yes ;;
+	    all) maybe_biarch=yes ;;
 	esac
 	case ${target} in
 	    powerpc64*-*-linux*spe* | powerpc64*-*-linux*paired*)
-- 
1.9.4

